test/cmd/policy.sh

```diff
@@ -58,19 +58,19 @@ os::cmd::expect_success_and_not_text 'oadm policy who-can create builds/docker'
 os::cmd::expect_success_and_not_text 'oadm policy who-can create builds/custom' 'namespaced-user'
 os::cmd::expect_success_and_not_text 'oadm policy who-can create builds/source' 'namespaced-user'
 os::cmd::expect_success_and_not_text 'oadm policy who-can create builds/jenkinspipeline' 'namespaced-user'
+os::cmd::expect_success_and_not_text 'oadm policy who-can create builds/custom' 'system:authenticated'
 os::cmd::expect_success_and_text     'oadm policy who-can create builds/docker' 'system:authenticated'
-os::cmd::expect_success_and_text     'oadm policy who-can create builds/custom' 'system:authenticated'
```

Comment:
Custom builds are no longer like any of the other build types, the comments over the blocks of tests no longer apply. Group the custom build tests together and comment them separately